src/gpb_names.erl

```diff
@@ -477,6 +486,18 @@ enum_renamings(PkgByProto, PkgRenamings, Defs, RenameOps) ->
          end
          || {{enum_containment, Proto}, EnumNames} <- Defs])).
 
+enum_field_renamings(PkgByProto, _PkgRenamings, Defs, RenameOps) ->
```

Comment:
You are not using the second argument? Why declare it?

Reply:
done.